src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -88,6 +88,8 @@ sealed trait Task {
     else
       Some(agentInfo.host)
   }
+
+  def version: Option[Timestamp]
```

Comment:
It's an option because `Task.Reserved` currently has no version (although it should). I didn't want to change that and introduce a bunch more changes therefore I added this for convenience getter.

Should be fixed and removed in #3867